GNUmakefile

```diff
@@ -124,7 +130,7 @@ pkg/windows_386/nomad: $(SOURCE_FILES) ## Build Nomad for windows/386
 
 pkg/windows_amd64/nomad: $(SOURCE_FILES) ## Build Nomad for windows/amd64
 	@echo "==> Building $@ with tags $(GO_TAGS)..."
-	@CGO_ENABLED=1 GOOS=windows GOARCH=amd64 \
+	@CGO_ENABLED=$(NON_LINUX_CGO_ENABLED) GOOS=windows GOARCH=amd64 \
```

Comment:
Despite `CGO_ENABLED=1` here, when we cross compiled, go was making a static binary, as I inspected the binary in a Windows machine with `dumpbin`.  I think (but not 100% certain) that we would have needed to set CC and CXX to configure the cross compiler for go to use.